app/gui/src/ide/initializer.rs

```diff
@@ -109,19 +106,16 @@ impl Initializer {
         match &self.config.backend {
             ProjectManager { endpoint } => {
                 let project_manager = self.setup_project_manager(endpoint).await?;
-                let project_name = self.config.project_name.clone();
-                let controller = controller::ide::Desktop::new_with_opened_project(
-                    project_manager,
-                    project_name,
-                )
-                .await?;
+                let maybe_project_name = self.config.project_name.clone();
+                let controller =
+                    controller::ide::Desktop::new(project_manager, maybe_project_name).await?;
```

Comment:
Are you able to refactor this line in such a way that it is not broken into 2 lines? You can create some additional variables if needed. Our code style tells about a strong preference towards single line expressions over multi-line ones.